examples/connectivity/plot_extract_regions_dictlearn_maps.py

```diff
@@ -116,21 +120,22 @@
 
 ################################################################################
 # Showing Default Mode Network (DMN) regions before and after region extraction
-# by manually identifying the index of DMN in ICA decomposed components
+# by manually identifying the index of DMN in Dictionary Learning decomposed
+# components
 from nilearn._utils.compat import izip
```

Comment:
I know that this is not part of your PR, but I should we should replace 'izip', that we are importing from a private module (bad, bad), with 'zip', which does almost the same thing and is a builtin (ie doesn't have to be imported).